src/libpgmoneta/utils.c

```diff
@@ -1488,6 +1490,31 @@ pgmoneta_delete_file(char* file)
    return ret;
 }
 
+int
```

Comment:
If it is non-static then it must be called `pgmoneta_XYZ`

Reply:
done and moved to `restore.c`